openedx/core/djangoapps/plugins/constants.py

```diff
@@ -76,3 +76,19 @@ class PluginSignals(object):
 
     RELATIVE_PATH = u'relative_path'
     DEFAULT_RELATIVE_PATH = u'signals'
+
+
+class PluginContextsViews(object):
+    STUDENT_DASHBOARD = u'student_dashboard'
```

Comment:
We may want to jump to using the Domain name (business concept) for this feature instead: `Course Dashboard`.  This is the name we're using in the [Ownership sheet](https://docs.google.com/spreadsheets/d/1qpWfbPYLSaE_deaumWSEZfz91CshWd3v3B7xhOk5M4U/edit#gid=1333824381), which we'll want to propagate overtime into the codebase as well.

Terminologies:

- Student -> Learner
- Dashboards: Course, Program, Instructor, Enterprise Admin, ...

Reply:
Fixed!